tools/src/edm4hep2json.cxx

```diff
@@ -0,0 +1,97 @@
+// EDM4hep
+#include "edm4hep2json.hxx"
+
+// ROOT
+#include "podio/ROOTReader.h"
+
+// *nix
+#include <getopt.h>
+
+void printHelp() {
+  std::cout << "Usage:\n"
+            << "  -i/--in-file            input file path\n"
```

Comment:
Not sure if `getopt` supports this, but this seems to be the only required input parameter, can it be made to one that does not require a flag? i.e. usage should look something like

`edm4hep2json [flag-args] inputfile`